app/main.py

```diff
@@ -1,8 +1,65 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(
+            self,
+            comfort_class: int,
+            clean_mark: int,
+            brand: str
+    ) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(
+            self,
+            distance_from_city_center: int,
+            clean_power: int,
+            average_rating: float,
+            count_of_ratings: int
+    ) -> None:
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def calculate_washing_price(
+            self,
+            car: Car
+    ) -> float:
+
+        cost = (
+            car.comfort_class * (self.clean_power - car.clean_mark)
+            * self.average_rating / self.distance_from_city_center
+        )
+        return round(cost, 1)
+
+    def wash_single_car(
+            self,
+            car: Car
+    ) -> None:
+        if self.clean_power > car.clean_mark:
+            car.clean_mark = self.clean_power
+
+    def rate_service(
+            self,
+            rating: float
+    ) -> None:
+        self.average_rating = round(
+            (self.average_rating * self.count_of_ratings
+             + rating) / (self.count_of_ratings + 1), 1
+        )
+        self.count_of_ratings += 1
+
+    def serve_cars(
+            self,
+            cars: list[Car]
+    ) -> float:
+        income = 0.0
+        for car in cars:
+            if self.clean_power > car.clean_mark:
+                cost = self.calculate_washing_price(car)
+                if cost > 0:
+                    income += cost
```

Comment:
no need for `cost > 0` check
```suggestion
                income += self.calculate_washing_price(car)
```